Add JSoubs Advice

jsoubliere opened the pull request:
Sage wisdom

acodeinprogress commented:
Hey @jsoubliere,

This looks great so far! There's just one thing that we'll want to change:
It looks like you altered the CSS link in the main index.html file to use your personal stylesheet rather than our original stylesheet.

The original link was: <link rel="stylesheet" type="text/css" href="css/style.css" />
 
but you changed it to: <link rel="stylesheet" type="text/css" href="css/jsoubs-advice.css" />

We'll need to change it back to that original link and then add, commit, and push your changes. You'll want to make sure to make those changes on your experiment branch on your computer and then push to the experiment branch on GitHub.

Once you do that, we can get your pull request merged in.

xoxo,
Sharon

jsoubliere commented:
Thanks Sharon for the input!

I think I fixed this correctly.

Why I changed it:

 I didn't see the original style.css in my forked master css folder thus
why I changed the CSS link in the first place. Is it there and I'm just not
seeing it?

Thanks for the help!
Jason

On Wed, Feb 6, 2019 at 3:37 PM Sharon <notifications@github.com> wrote:

> Hey @jsoubliere <https://github.com/jsoubliere>,
>
> This looks great so far! There's just one thing that we'll want to change:
> It looks like you altered the CSS link in the main index.html file to use
> your personal stylesheet rather than our original stylesheet.
>
> The original link was:
>
> but you changed it to:
>
> We'll need to change it back to that original link and then add, commit,
> and push your changes. You'll want to make sure to make those changes on
> your experiment branch on your computer and then push to the experiment
> branch on GitHub.
>
> Once you do that, we can get your pull request merged in.
>
> xoxo,
> Sharon
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/skillcrush/105-practice/pull/229#issuecomment-461216282>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/As5CKkFnK7t0x8q9UwUUuz40MPhwnASAks5vK1k-gaJpZM4amL9m>
> .
>

acodeinprogress commented:
Hey @jsoubliere,

Thank for pointing that out! You're totally right - we'll get that added in.

As for your pull request, you did a great job! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).

xoxo,
Sharon
